src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
+  const ignorePhrases = [
+    "@presubmit ignore",
+    "@presubmit: ignore",
+    "@presubmit skip",
+    "@presubmit: skip",
+    "@presubmitai ignore",
+    "@presubmitai: ignore",
+    "@presubmitai skip",
+    "@presubmitai: skip",
+  ];
+  const bodyLower = (pull_request.body ?? "").toLowerCase();
+
+  for (const phrase of ignorePhrases) {
+    if (bodyLower.includes(phrase.toLowerCase())) {
+      info(`ignoring pull request because of '${phrase}' in description`);
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
This new functionality should be accompanied by unit tests to verify the behavior of `shouldIgnorePullRequest()`. Tests should cover various scenarios including:
- Different ignore phrase formats
- Case sensitivity
- Empty PR description
- PR description with whitespace around ignore phrases
- PR description without any ignore phrases